auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -207,6 +216,11 @@ public FirebaseUiException getError() {
         return mException;
     }
 
+    @Nullable
+    public AuthCredential getCredentialForLinking() {
+        return mPendingCredential;
```

Comment:
Is it still going to be valid after we use it?

Reply:
Yes.